src/skmultiflow/trees/nodes/active_learning_node_perceptron.py

```diff
@@ -102,9 +102,9 @@ def update_weights(self, X, y, learning_ratio, rht):
         """
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```

Comment:
The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.